app/javascript/src/components/Dashboard/Contacts/Table/ContactDropdown.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { Dropdown } from "neetoui";
+
+import DropdownMenu from "./DropdownMenu";
+
+const ContactDropdown = ({ handleDelete }) => {
+  const { Menu, MenuItem } = Dropdown;
+  return (
+    <Dropdown customTarget={DropdownMenu}>
+      <Menu>
+        <MenuItem.Button onClick={handleDelete}>Delete</MenuItem.Button>
```

Comment:
https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663844699333179?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

Reply:
Done. This was corrected when you mentioned it in the previous PR. Thanks for pointing this out.